doc/classes/Camera2D.xml

```diff
@@ -324,18 +324,24 @@
 	</methods>
 	<members>
 		<member name="anchor_mode" type="int" setter="set_anchor_mode" getter="get_anchor_mode" enum="Camera2D.AnchorMode">
+			The Camera2D's anchor point. See [code]ANCHOR_MODE_*[/code] constants.
 		</member>
 		<member name="current" type="bool" setter="_set_current" getter="is_current">
+			If [code]true[/code] this camera is the active camera for the current scene. Only one camera can be current, so setting a different camera [code]true[/code] will disable this one.
```

Comment:
This is more accurate:

    setting a different camera [code]current[/code] will disable this one.